integration_test/third_party_apps_test.go

```diff
@@ -73,6 +73,19 @@ func osFolder(platform string) string {
 	return "linux"
 }
 
+// rejectDuplicates looks for duplicate entries in the input slice and returns
+// an error if any is found.
+func rejectDuplicates(apps []string) error {
+	unique := make(map[string]bool)
```

Comment:
The name `unique` implies that it would be true for items that are unique, and false for duplicates. Would `seen` be a better name?